.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,6 @@
+Fixes #.
```

Comment:
Should we have more sections?

I think having sections for the "purpose of the pull request", "summary of the changes", "Usage/example of changes", etc. are helpful to reviewers.

Reference:
https://github.com/blog/1943-how-to-write-the-perfect-pull-request

Reply:
Good suggestion. Added some in 243e69c